src/monkeyflocker/client.py

```diff
@@ -67,25 +67,25 @@ async def start(self, spec_file: Path) -> None:
         assert self._session, "Must be used as a context manager"
         with spec_file.open("r") as f:
             spec = yaml.safe_load(f)
-        self._logger.info("Starting flock %s", spec["name"])
+        self._logger.info(f"Starting flock {spec['name']}")
         url = urljoin(self._base_url, "/mobu/flocks")
         await self._session.put(url, json=spec)
-        self._logger.info("Flock %s started", spec["name"])
+        self._logger.info(f"Flock {spec['name']} started")
```

Comment:
Why these changes? I was using `%s` formatting to avoid the nested quote types.

Reply:
When I run the `%s` version in my development environment I get an error about wrong number of arguments.  Maybe the logger changes between Python 3.8 and 3.9 or something, and I am running one and you're running the other?

Reply:
Oh, weird. Okay, in that case, looks good to me.